l10n.toml

```diff
@@ -123,7 +164,3 @@ branches = [
 [[paths]]
   reference = "components/feature/addons/src/main/res/values/strings.xml"
   l10n = "components/feature/addons/src/main/res/values/values-{android_locale}/strings.xml"
-
-[[paths]]
-  reference = "components/feature/awesomebar/src/main/res/values/strings.xml"
-  l10n = "components/feature/awesomebar/src/main/res/values-{android_locale}/strings.xml"
```

Comment:
@Pike any reason why we are removing `feature-awesomebar`?

Reply:
Sorry, didn't check the open PR to android-l10n when reviewing the locales changes there, and now we have a chicken and an egg.

Let me try to push a follow-up here.

Reply:
👍